Demo/SwiftUI/Info.plist

```diff
@@ -43,6 +43,8 @@
 	<string>1</string>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
+	<key>NSPhotoLibraryAddUsageDescription</key>
```

Comment:
Also need same thing for HostUI target?

Reply:
The UIKit host already have this in a previous PR.